src/quantum/tp6-32bit.js

```diff
@@ -0,0 +1,89 @@
+import _ from 'lodash';
+import React from 'react';
+import { withStyles } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { PAGES } from './config';
+import DashboardPage from '../components/DashboardPage/index';
+import PerfherderGraphContainer from '../containers/PerfherderGraphContainer/index';
+
+
+const styles = {
+  // This div helps with canvas size changes
+  // https://www.chartjs.org/docs/latest/general/responsive.html#important-note
+  body: {
+    backgroundColor: 'white',
+  },
+  area: {
+    display: 'flex',
+  },
+  column: {
+    flex: '50%',
+    alignItems: 'center',
+  },
+  chart: {
+    justifyContent: 'center',
+    padding: '1rem',
+
+  },
+
+};
+
+const TP6_32 = ({ classes }) => {
+  const allCharts = _
+    .chain(PAGES.data)
+    // ZIP HEADER WITH ROWS TO GET OBJECTS
+    .map(row => _.zipObject(PAGES.header, row))
+    // GROUP BY title
+    .groupBy(row => row.title)
+    // LOOP OVER EACH KEY/VALUE
+    .toPairs()
+    .map(([title, series], _i) => {
+      return (
+        <div className={classes.chart}>
+          <PerfherderGraphContainer
+          // key={`pages${i}`}
+            title={title}
+            series={
+            _
+              .chain(series)
+              .map((s) => {
+                return {
+                  label: s.label,
+                  seriesConfig: s,
+                };
+              })
+              .value()
+          }
+          />
+        </div>
+      );
+    })
+    // SPLIT INTO TWO
+    .chunk(2)
+    // TRANSPOSE, SO WE HAVE TWO COLUMNS
+    .unzip()
+    .value();
+
+  return (
+    <div className={classes.body}>
+      <DashboardPage title='TP6 - Page load on 32bit'>
+        <div className={classes.area}>
+          <div className={classes.column}>
```

Comment:
What you're doing here is a bit hard to understand.
I use a `flex` container you can let the page determine if we can fit 2 or 3 or even 4 items per row.

This is also what happens when not using `flex` (see the cut out on the right side):
<img width="1121" alt="image" src="https://user-images.githubusercontent.com/44410/49877226-f3dc1580-fdf2-11e8-80d1-42a825a796ec.png">

Reply:
cool, but I do not know how to get flex to decide the number of columns. 

I tried regular flow layout (`display: inline`)  but that looks like garbage: As the charts load, the page is resizing too many time.

Reply:
resolving this because I will try a `grid` solution